charts/gateway/README.md

```diff
@@ -1157,6 +1158,51 @@ The full default is this
     com.l7tech.server.clusterStaleNodeCleanupTimeoutSeconds=86400
     # Additional properties go here
 ```
+### DISKLESS Configuraton
+DISKLESS_CONFIG is a flag to pass gateway sensitive data to be mounted to the container gateway file system or passed as environment variables.
+By Default, true, environment variables are used to configure Gateway.
+
+When DISKLESS_CONFIG is false , Gateway will be configured from node.properties
+
+#### node.properties
+Note: 
+- Database configuration should be same as mentioned in node.properties
+```
+node.cluster.pass=newpassword
+admin.user=admin
+admin.pass=newpassword
+node.db.config.main.host=myDBHost.com
+node.db.config.main.port=3306
+node.db.config.main.name=ssg
+node.db.config.main.user=gateway
+node.db.config.main.pass=newpassword
+```
+- For derby database, it is required to add ***node.db.type=derby*** to node.properties
+```
+node.cluster.pass=newpassword
+admin.user=admin
+admin.pass=newpassword
+node.db.type=derby
+node.db.config.main.user=gateway
+```
+
+- Mounting a pre-configured node.properties to container gateway
+  - Referenced via existingGatewaySecretName
+  - In following example , pre-existing secret is  node-properties-secret
```

Comment:
We need to add some lines here regarding examples:
1. Create new secret done in #5 
2. Use existing secret